crates/storage-ops/src/rollback/node/fullnode.rs

```diff
@@ -200,12 +201,14 @@ impl FullNodeLedgerRollback {
     ) -> Result {
         let mut batch = SchemaBatch::new();
 
-        let pending_sequencer_commitments = self
+        let mut pending_sequencer_commitments = self
             .ledger_db
             .iter_with_direction::<PendingSequencerCommitments>(
                 Default::default(),
                 ScanDirection::Backward,
             )?;
+        pending_sequencer_commitments.seek_to_last();
```

Comment:
This looks a little bit awkward. There is RawDbReverseIterator, which does it, so I think for Forward we have to return a ForwardIterator, while Backward has to be RawDbReverseIterator inside, so we don't need to manually seek_to_last.

Also I believe their impl of forward/backward iterators is faulty because RawDbReverseIterator::seek must use seek_for_prev (and they do), but in common version they don't check for it.